utils.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Apr 16 19:12:51 2023
+
+@author: Sumedh Prasad
+
+utils code
+"""
+
+from typing import List, Tuple
+import os
+from os.path import join, dirname
+import logging
+
+from pathlib import Path
+import mysql.connector as mysql
+from dotenv import load_dotenv,dotenv_values
+
+os.chdir(Path("E:/E2E_DTP_Project"))
```

Comment:
on line 19, you don't need `os.chdir()`, just `Path(..)` should be fine

Reply:
you actually don't even need this line of line

Reply:
ok I will remove it now from the code

Reply:
Remove completed